### PR TITLE
prctl SME flags for gnu arm64.

### PR DESCRIPTION
**Pull request by devnexen**

### CODE REVIEW

**Review by JohnTitor — APPROVED** [review]

File: libc-test/build.rs [hunk 1]
```diff
@@ -3583,6 +3583,8 @@ fn test_linux(target: &str) {
 
             // present in recent kernels only
             "PR_PAC_SET_ENABLED_KEYS" | "PR_PAC_GET_ENABLED_KEYS" => true,
+            // present in recent kernels only
+            "PR_SME_SET_VL" | "PR_SME_GET_VL" | "PR_SME_VL_LEN_MAX" | "PR_SME_SET_VL_INHERIT" | "PR_SEM_SET_VL_ONE_EXEC" => true,
```
Comment (JohnTitor) [inline libc-test/build.rs:3587]: Could you add a version number and toolchain name?

File: libc-test/build.rs [hunk 2]
```diff
@@ -3586,8 +3586,10 @@ fn test_linux(target: &str) {
             // present in recent kernels only
             "PR_SCHED_CORE" | "PR_SCHED_CORE_CREATE" | "PR_SCHED_CORE_GET" | "PR_SCHED_CORE_MAX" | "PR_SCHED_CORE_SCOPE_PROCESS_GROUP" | "PR_SCHED_CORE_SCOPE_THREAD" | "PR_SCHED_CORE_SCOPE_THREAD_GROUP" | "PR_SCHED_CORE_SHARE_FROM" | "PR_SCHED_CORE_SHARE_TO" => true, 
 
-            // present in recent kernels only
+            // present in recent kernels only >= 5.13
             "PR_PAC_SET_ENABLED_KEYS" | "PR_PAC_GET_ENABLED_KEYS" => true,
+            // present in recent kernels only >= 5.19
+            "PR_SME_SET_VL" | "PR_SME_GET_VL" | "PR_SME_VL_LEN_MAX" | "PR_SME_SET_VL_INHERIT" | "PR_SEM_SET_VL_ONE_EXEC" => true,
```
Comment (valpackett) [inline libc-test/build.rs:3592]: typo, `s/_SEM_/_SME_/`

**Pull request merged**